src/main/java/uk/gov/hmcts/divorce/caseworker/service/IssueApplicationService.java

```diff
@@ -38,9 +38,13 @@ public CaseData aboutToSubmit(final CaseData caseData,
             .userAuthToken(idamAuthToken)
             .build();
 
-        return caseDataUpdaterChainFactory
+        CaseData updatedCaseData = caseDataUpdaterChainFactory
             .createWith(caseDataUpdaters)
             .processNext(caseDataContext)
             .getCaseData();
+
+        updatedCaseData.setIssueDate(LocalDate.now());
```

Comment:
To test this, you could use the Clock...
   @Autowired
    private Clock clock;
and then use
LocalDate.now(clock)

Reply:
done here https://github.com/hmcts/nfdiv-case-api/pull/374/commits/0cc90160842c6639cc8f7af4d120ca3de04d7eab